pages/index.vue

```diff
@@ -2,7 +2,7 @@
   <div> <Cards v-bind:articles="articles"/></div>
```

Comment:
here again 
``` suggestion 
 <div> <Cards :articles="articles"/></div>
 ```
 but again importing only component in a page is not a good practice, follow the approach that I shared on DM